server/src/test/java/server/haengdong/presentation/BillActionControllerTest.java

```diff
@@ -66,4 +71,26 @@ void saveAllBillAction1() throws Exception {
                 .andDo(print())
                 .andExpect(status().isBadRequest());
     }
+
+    @DisplayName("지출 내역을 삭제한다.")
+    @Test
+    void deleteBillAction() throws Exception {
+        String token = "TOKE11";
+
+        mockMvc.perform(delete("/api/events/{token}/actions/{actionId}/bills", token, 1))
```

Comment:
```suggestion
        mockMvc.perform(delete("/api/events/{eventId}/actions/{actionId}/bills", token, 1))
```
c: API 명세와 PathParameter가 달라서 남겨요!

Reply:
다른 컨트롤러 테스트 보니까 대부분 events 뒤에 token이라고 명시되어 있어서 바꿀 거라면 아예 전체적인 수정이 필요할 것 같아요